craft_parts/parts.py

```diff
@@ -848,6 +857,14 @@ def default_partition(self) -> str:
             return self._partitions[0]
         return DEFAULT_PARTITION
 
+    @property
+    def bootstrap_overlay(self) -> bool:
```

Comment:
```suggestion
    def is_bootstrap_overlay(self) -> bool:
```

Makes it clearer that it's a query, and not something else